src/classes/HealthCheckDefinitionMapper_TEST.cls

```diff
@@ -0,0 +1,81 @@
+    * HealthCheckDefinitionMapper class when one does not already exist.
+    ***************************************************************************************************************************/
+    @isTest
+    private static void getInstanceNew() {
+        Test.startTest();
+        HealthCheckDefinitionMapper healthCheckDefinitionMapperInstance = HealthCheckDefinitionMapper.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(HealthCheckDefinitionMapper.instance, healthCheckDefinitionMapperInstance, 'Instance of service class returned should match static instance variable.');
+        System.assertEquals(true, healthCheckDefinitionMapperInstance != null, 'Instance of service class should not be null.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getInstance method returns the existing instance of the 
+    * HealthCheckDefinitionMapper class when one already exists.
+    ***************************************************************************************************************************/
+    @isTest
+    private static void getInstanceExisting() {
+        Test.startTest();
+            HealthCheckDefinitionMapper healthCheckDefinitionMapperInstance1 = HealthCheckDefinitionMapper.getInstance();
+            HealthCheckDefinitionMapper healthCheckDefinitionMapperInstance2 = HealthCheckDefinitionMapper.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(HealthCheckDefinitionMapper.instance, healthCheckDefinitionMapperInstance1, 'Instance of service class returned should match static instance variable.');
+        System.assertEquals(healthCheckDefinitionMapperInstance1, healthCheckDefinitionMapperInstance2, 'Subsequent retrievals of service class instance should return existing instance.');
+        System.assertEquals(true, healthCheckDefinitionMapperInstance1 != null, 'Instance of service class should not be null.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getHealthCheckDefinitionModels method in Health Check Mapper class
+    * returns a list of Health Check Definition models
+    ***************************************************************************************************************************/
+    @isTest
+    private static void getHealthCheckDefinitionModelsExistingFromMapper() {
+        Test.startTest();
+            List<HealthCheckDefinitionModel> healthCheckDefinitionModels = HealthCheckDefinitionMapper.getInstance().getHealthCheckDefinitionModels();
+        Test.stopTest();
+
+        System.assertEquals(3, healthCheckDefinitionModels.size());
+        //TODO: Update assert once HC models aren't hardcoded
```

Comment:
Good note to leave here. We'll probably have to update this every major PR until we solve for persistence.